internal/testing/k8s-shredder.yaml

```diff
@@ -16,13 +16,28 @@ spec:
       labels:
         app: k8s-shredder
     spec:
+      affinity:
+        nodeAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
```

Comment:
This might be a probability, we can enforce it with nodeselector.

Reply:
I know, I intentionally put preferred in order to allow a bit f flexibility. I could also have used `requiredDuringSchedulingIgnoredDuringExecution` instead of `nodeselector` btw.